tests/behaviour/contracts/expressions/literalExpressionEvaluator.sol

```diff
@@ -0,0 +1,17 @@
+pragma solidity ^0.8.10;
+
+//SPDX-License-Identifier: MIT
+
+contract WARP {
+  function false_and() pure public returns (bool) {
+    return callNull() && false;  // false 
+  }
+
+  function true_or() pure public returns (bool) {
+    return callNull() || true;  // true 
+  }
+
+  function callNull() pure public returns (bool) {
+    return true && false;
+  }
```

Comment:
These functions are still not checking the behavior you implemented. Notice in the case of the `&&` operation that one of the members is `false`, so the entire expression is `false` (that's the logic that was already implemented in the pass); the behavior you added is for the case where that member is `true`, so the expression can be replaced from `callNull() && true` to just `callNull()`. Same thing about the `||` operation.